travel-event-host-app/src/components/auth-dialog/AuthDialog.tsx

```diff
@@ -52,16 +55,45 @@ export default function AuthDialog(props: AuthDialogProps) {
         return;
       }
       // TODO: submit the form
-      console.log('Signup form is valid');
+      const fetch = async () => {
```

Comment:
Rename this method so it doesn't collide with the fetch API and is more descriptive